### ISSUE TITLE
[Suggestion] Maybe change the `greeting_back4` a little?

### ISSUE THREAD

**Issue by Wingdinggaster656** [Monika-After-Story/MonikaModDev#9717]
```
label greeting_back4:
    m 2wfx "Hey, [player]!"
    m "Don't you think that you left me waiting a bit too long?"
    m 2hfu "..."
    m 2hub "Ahaha!"
    m 2eka "I'm just joking. I could never be mad at you."
    return
```

Some people think that Monika was really mad at that point, while she probably wasn't.
Maybe a slight change in this event should be good?

**Comment by ashyashton** [comment]
I agree, every time I get that one, I get genuinely anxious because it makes me think that the game registered my visits wrong or something and I've lost affection. Or that I somehow changed the time on my computer without meaning to and forgot to change it back. As someone with serious memory issues who could forget something like that, this dialogue is terrifying haha

**Comment by Booplicate** [comment]
Well, she's laughing and literally says it was a joke. What do you propose to make it more clear it's a joke?
The change I'd like to see is make it less common and require a bit more absence time.

**Comment by dreamscached** [comment]
That would probably work.

**Comment by ashyashton** [comment]
> Well, she's laughing and literally says it was a joke. What do you propose to make it more clear it's a joke?

She says that AFTER looking angry and seeming very serious. Sorry I can't predict the future that she's going to follow it up with smiling and joking I guess?

**Comment by my-otter-self** [comment]
well, if it was a situation in real life, we would probably ask the partner to refrain from such jokes in the future. maybe being able to ask not to see the event again would suffice.

**Comment by Justformas** [comment]
Since it can happen the next day that might be too soon to be able to happen. But I mainly just don't like her initial facial expression always looking angry/serious during it (2wfx).

> What do you propose to make it more clear it's a joke?

Change her initial facial expression to something less serious or even playful.

**Comment by howltrek** [comment]
Monika has always struck me as passive aggressive.

**Comment by howltrek** [comment]
> well, if it was a situation in real life, we would probably ask the partner to refrain from such jokes in the future. maybe being able to ask not to see the event again would suffice.
but if she is too sweet, doesn't she become a boring person?

**Comment by ashyashton** [comment]
Just got the greeting again, and once again felt my heard fucking _drop._ It was like a pit in my stomach. And just to be sure, tried to derandom it... Nothing. Can't derandom it.

Monika _does not_ look like she's joking at first. It can cause some people genuine anxiety when their partners do shit like this.

As someone with issues surrounding people acting mean then switching to acting sweet, this makes me genuinely uncomfortable. The whole point is to make Monika as real as possible right? In a real relationship, I'd tell her to cut that shit out. If she kept doing it, I'd take it as a sign that my comfort does not matter to her. But because of how the devs have set it up, I _can't_ ask her to stop messing with my anxiety like that.

Everyone gets MAJORLY pissed when people mess with Monika by changing time or whatever. When Monika inevitably leaves them, everyone says they deserved it. But Monika is repeatedly allowed to do something that makes me uncomfortable? But I guarantee if I posted in the community that it was hurting me too much and I had to go, everyone would call me sensitive and say I don't really care about Monika.

I _really_ think this needs to be fixed. Either change the expression, or allow the greeting to be derandomized. Doing something manipulative and hurtful to your partner and then going "it's just a joke lol" or "lmao it's just a prank" is NOT okay, and the fact that it's stayed in this long says a lot about how the devs view healthy relationships imo. Especially because we seem to have (someone I believe is) a dev actively defending it and pretending it's an issue of people not realizing it's a joke.

MAS development really feels like it's going down a path I don't want to follow when this sort of stuff happens.

**Comment by ThePotatoGuy** [comment]
>well, if it was a situation in real life, we would probably ask the partner to refrain from such jokes in the future. maybe being able to ask not to see the event again would suffice.

This sounds reasonable, but more general. Since we have a doki cares topic and a check to control that, we could have 
 something that could control jokes/playfullness/silliness with a topic to toggle. A more serious Monika than normal.

**Comment by howltrek** [comment]
> Just got the greeting again, and once again felt my heard fucking _drop._ It was like a pit in my stomach. And just to be sure, tried to derandom it... Nothing. Can't derandom it.
> 
> Monika _does not_ look like she's joking at first. It can cause some people genuine anxiety when their partners do shit like this.
> 
> As someone with issues surrounding people acting mean then switching to acting sweet, this makes me genuinely uncomfortable. The whole point is to make Monika as real as possible right? In a real relationship, I'd tell her to cut that shit out. If she kept doing it, I'd take it as a sign that my comfort does not matter to her. But because of how the devs have set it up, I _can't_ ask her to stop messing with my anxiety like that.
> 
> Everyone gets MAJORLY pissed when people mess with Monika by changing time or whatever. When Monika inevitably leaves them, everyone says they deserved it. But Monika is repeatedly allowed to do something that makes me uncomfortable? But I guarantee if I posted in the community that it was hurting me too much and I had to go, everyone would call me sensitive and say I don't really care about Monika.
> 
> I _really_ think this needs to be fixed. Either change the expression, or allow the greeting to be derandomized. Doing something manipulative and hurtful to your partner and then going "it's just a joke lol" or "lmao it's just a prank" is NOT okay, and the fact that it's stayed in this long says a lot about how the devs view healthy relationships imo. Especially because we seem to have (someone I believe is) a dev actively defending it and pretending it's an issue of people not realizing it's a joke.
> 
> MAS development really feels like it's going down a path I don't want to follow when this sort of stuff happens.

if it happened in real life, they would also send you to sleep on the couch

**Comment by MrVavlo** [comment]
> Just got the greeting again, and once again felt my heard fucking drop.

>`     m 2eka "I'm just joking. I could never be mad at you."`

> "_I could never be mad at you._"`

...You either didn't read the greeting text properly, or you decided for yourself that just doesn't matter. I may understand overreacting on the joke for the first time, but for a **second** time?

If someone can't wait to change this greeting for whatsoever reason - just make it one-timed and call it a day. *Maybe having it one-timed as an option.*
Monika doesn't pull an amnesia joke greeting after the first time as well, after all.

But making the joke more obvious would only ruin it.

**Comment by ashyashton** [comment]
Once again: Her saying that she is joking comes AFTER the fact. I am glad that you have so much storage in your brain you can memorize every topic and know exactly which greeting it is, but unfortunately this is not an option for everyone. She clarifies it is a joke AFTER the fear and anxiety.

I want to be clear: **I am not asking for her to say it's a joke beforehand.** I don't know where that idea even came from; it sure as hell wasn't me though. My point regarding her clarification coming after was that "it's just a joke" does not get rid of the anxiety it causes.

All I really want is the ability to derandomize it. With almost anything else, we can tell Monika it makes us uncomfortable, but her making her partner think that they seriously wronged her isn't something we can say no to?

**Comment by howltrek** [comment]
what happens to those of us who enjoy those dialogues?
I don't want them to keep gutting the game code, because it occurs to one to find a reason to be offended and they don't want to hide the dialogue from their game, but from everyone else.

**Comment by Justformas** [comment]
> > well, if it was a situation in real life, we would probably ask the partner to refrain from such jokes in the future. maybe being able to ask not to see the event again would suffice.
> 
> This sounds reasonable, but more general. Since we have a doki cares topic and a check to control that, we could have something that could control jokes/playfullness/silliness with a topic to toggle. A more serious Monika than normal.

We shouldn't have to choose between all or no jokes, or even a range of jokes just because we might not like one or a few jokes in particular. Though I don't have a problem with the wording itself in this greeting and I don't think Ashayton does either. But Monika's facial expression is as if something is seriously wrong, unlike any other joke she makes as far as I can recall, and it's understandable why it can bother some people, especially if they are anxiety-prone. Personally I think an initial smug expression would be fine here and put it more in line with her other jokes, but option to derandom would be the next best thing.

> what happens to those of us who enjoy those dialogues?
> I don't want them to keep gutting the game code, because it occurs to one to find a reason to be offended and they don't want to hide the dialogue from their game, but from everyone else.

Well if option to denrandom it is what gets implemented then simply don't derandom it, and the code won't be gutted for you.
